api/bootstrap.proto

```diff
@@ -73,6 +73,8 @@ message Bootstrap {
       // <arch_overview_dynamic_config_sds>`.
       ConfigSource sds_config = 1;
     }
+
+    // [#not-implemented-hide:]
```

Comment:
Why is this now hidden? Shoudl we tag `deprecated_v1` with a `deprecated` proto annotation?

Reply:
AFAICT this is only used during v1 -> v2 conversion. We hide similar cases as a normal user would not use this.

Reply:
OK, I think we should also add the `deprecated` annotation to these proto fields, can you sneak that into this PR?

Reply:
will do